pytest_rts/tests/test_git.py

```diff
@@ -0,0 +1,54 @@
+"""Unittests for Git functionality"""
+
+from typing import List
+
+import pytest
+
+from pytest_rts.tests.fake_diffs import FAKE_DIFF_1, FAKE_DIFF_2, FAKE_DIFF_3
```

Comment:
Are those `FAKE_DIFF_*` used anywhere else? I would maybe move it here to have data and tests in the same file. Also, do we really need such a big diff? Is it possible to remove "duplicated cases" from there?

Reply:
I just left one diff now inside the testcase. It should have the same cases but simplified.